resources/js/components/salary/AdminDashboard.vue

```diff
@@ -0,0 +1,124 @@
+                            <button class="btn btn-default">Search</button>
+                            <button v-on:click="clearSearch()" class="btn btn-default">Reset</button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div style="clear:both"></div>
+            <div class="row">
+                <div class="col-md-12">
+                    <table class="table table-bordered">
+                        <thead>
+                            <tr>
+                                <th rowspan="2">ID</th>
+                                <th rowspan="2">Name</th>
+                                <th rowspan="2">Work?</th>
+                                <th colspan="4">Work Counter</th>
+                                <th rowspan="2">Current Fund</th>
+                            </tr>
+                            <tr>
+                                <th>Offline</th>
+                                <th>Morning</th>
+                                <th>Afternoon</th>
+                                <th>Fulltime</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr v-if="!searchResult.length">
+                                <td colspan="100">No user found</td>
+                            </tr>
+                            <tr v-for="user, index in searchResult">
+                                <td>{{user.id}}</td>
+                                <td>{{user.name}}</td>
+                                <td>
+                                    <a v-on:click="checkpoint(user.id,0)" class="btn btn-sm btn-danger">OFFLINE</a>
+                                    <a v-on:click="checkpoint(user.id,1)" class="btn btn-sm btn-info">MORNING</a>
+                                    <a v-on:click="checkpoint(user.id,2)" class="btn btn-sm btn-info">AFTERNOON</a>
+                                    <a v-on:click="checkpoint(user.id,3)" class="btn btn-sm btn-success">FULLTIME</a>
+                                </td>
+                                <td>{{user.offline}}</td>
+                                <td>{{user.morning}}</td>
+                                <td>{{user.afternoon}}</td>
+                                <td>{{user.fulltime}}</td>
+                                <td>$ {{user.fund}}</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+<script>
+export default {
+    data(){
+        return{
+            users: [],
+            searchInput: {
+                content: ""
+            },
+            searchResult: []
+        }
+    },
+    mounted() {
+        var app = this;
+        axios.get(laroute.route('api.salary.today'))
+            .then(resp => {
+                app.users = resp.data;
```

Comment:
đã dùng arrow function thì ko cần app = this r gọi kiểu này nữa. gọi this.users luôn.
tìm hiểu thêm về arrow function nhé.